### ISSUE TITLE
can u add this to readme.md for python users to help them make secrets in replit for the bot token?

### ISSUE THREAD

**Issue by TheCodingPsn** [IMarkoMC/Tokens#82]
@IMarkoMC https://www.youtube.com/watch?v=08Y2Hqf1e-c

**Comment by TheCodingPsn** [comment]
btw keep this up to shit on the clowns to blatantly put their tokens in the web (I used to be one lol)

**Comment by IMarkoMC** [comment]
We already have guides in how to use ENV variables [here](https://github.com/IMarkoMC/Tokens/tree/main/how_to_use_env) 

Text format is better for this because there's no way someone skips past a important part, feel free to open a PR if you think something can be worded differently or if you have some changes to propose

**Comment by IMarkoMC** [comment]
The guides for Replit stuff are [this](https://github.com/IMarkoMC/Tokens/blob/main/how_to_use_env/creating_environments_variables/replit.md) and [this](https://github.com/IMarkoMC/Tokens/blob/main/how_to_use_env/use_variable_in_code/python.md)

Adding something like a readme to the main folder (how_to_use_env) might be a good idea and i might do it later to make searching the guides easier

**Comment by LiteEagle262** [comment]
if you need anymore help here is a example I made in 2 secs https://hastebin.com/oqugoyidit.sql

**Comment by IMarkoMC** [comment]
Closing this due to inactivity.

**Comment by skidmaster69420** [comment]
I just noticed I haven't responded to this message yet, and I am legally required to reply to all of them. Thanks for your suggestion for improving Comrade Marko's repository. 

P.S. As for the fly in my room? I have captured him successfully. He is now my captive. I found out the solution is to go slow and steady, but I got him so bad. He's mine now. But the paper I used was too short, so I used some "cease and desist" notices laying around. I lied about being peaceful. That fucking fly has been keeping me up every night. He's going to sit there and suffocate in that glass. I hate that damn fly.
